### PR TITLE
Added Monkey Command to send a monke gif.

### PR DESCRIPTION
**Pull request by Solastis**
[contributing]: https://github.com/Ree6-Applications/Ree6/blob/master/.github/CONTRIBUTING.md

## Pull Request Etiquette

- [x] I have checked the PRs for upcoming features/bug fixes.
- [x] I have read the [contributing guidelines][contributing].

### Changes

- [x] Internal code
- [ ] Documentation
- [ ] Other: \_____ 

Closes Issue: NaN

## Description

Replace this sentence with general description of what your Pull Request does.

### CODE REVIEW

File: src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import de.presti.ree6.commands.interfaces.Command;
+import de.presti.ree6.commands.interfaces.ICommand;
+import de.presti.ree6.main.Main;
+import de.presti.ree6.utils.data.Data;
+import de.presti.ree6.utils.external.RequestUtility;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.interactions.commands.build.CommandData;
+
+/**
+ * A command to get a random dog images.
```
Comment (DxsSucuk) [inline src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java:16]: Does not seem right.

File: src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+ */
+@Command(name = "monke", description = "Shows monke.", category = Category.FUN)
+public class Monkey implements ICommand {
+
+	/**
+	 * @inheritDoc
+	 */
+	@Override
+	public void onPerform(CommandEvent commandEvent) {
+		JsonObject js = RequestUtility.request(RequestUtility.Request.builder()
```
Comment (DxsSucuk) [inline src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java:26]: Sure that this code even works?

**Review by DxsSucuk — CHANGES_REQUESTED** [review]

File: src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java (hunk 2 above)
Comment (Solastis) [reply src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java:26]: no

**Review by DxsSucuk — CHANGES_REQUESTED** [review]
Code format looks weird?

**Comment by DxsSucuk** [comment]
> > ??????? Code format looks weird?
> 
> why

If look at the code you will see that there are unneeded spaces which lets it look a bit odd

File: src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+	public void onPerform(CommandEvent commandEvent) {
+
+		final EmbedBuilder em = new EmbedBuilder();
+
+		em.setTitle("Monke!");
+		em.setColor(BotWorker.randomEmbedColor());
+		em.setImage("https://c.tenor.com/Y89PE1f7exQAAAAd/reject-modernity-return-to-monke.gif");
+		em.setFooter("Requested by " + commandEvent.getMember().getUser().getAsTag() + " - " + Data.ADVERTISEMENT,
+		             commandEvent.getMember().getUser().getAvatarUrl()
+		);
```
Comment (DxsSucuk) [inline src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java:32]: This one for example is scuffed.

File: src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+		Main.getInstance().getCommandManager().sendMessage(em,
+		                                                   commandEvent.getChannel(),
+		                                                   commandEvent.getInteractionHook()
+		);
```
Comment (DxsSucuk) [inline src/main/java/de/presti/ree6/commands/impl/fun/Monkey.java:37]: The same goes for this call.

**Review by DxsSucuk — APPROVED** [review]

**Review by DxsSucuk — APPROVED** [review]

**Pull request merged**